CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -369,7 +369,7 @@
       </trans-unit>
       <trans-unit id="Credits Lists" xml:space="preserve">
         <source>Credits Lists</source>
-        <target state="translated">Credits enthalten …</target>
+        <target state="translated">Auswahl für Credits</target>
```

Comment:
"Credits enthalten ..." just didn't look and feel right. "Auswahl für Credits" means "Selection for Credits" which sacrifices the noun-verb-structure, but overall feels... slightly better here imho.